src/hooks/useInput.js

```diff
@@ -0,0 +1,9 @@
+import { useState } from 'react';
+
+export const useInput = (initialValue) => {
+  const [value, setValue] = useState(initialValue);
+  return [
```

Comment:
could you be so kind and just explain these hook to me? i understand what it does, but not step by step

Reply:
we can talk about it later, it is faster.

Reply:
allright